api/index.md

```diff
@@ -5,13 +5,570 @@ _navPath: toc.html
+   <td>
+   Provides thread-safe collection classes that should be used instead of the corresponding types in the System.Collections and System.Collections.Generic namespaces whenever multiple threads are accessing the collection concurrently. However, members accessed through extension methods or through the interfaces that these collection classes implement are not guaranteed to be thread safe and may need to be synchronized by the caller.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Collections.Generic.html">System.Collections.Generic</a></td>
+   <td>
+   Contains types that define generic collections, which allow users to create strongly-typed collections that provide better type safety and performance than non-generic collections.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Collections.ObjectModel.html">System.Collections.ObjectModel</a></td>
+   <td>
+   Contains classes that can be used as collections in the object model of a reusable library. Use these classes when properties or methods return collections.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Collections.Specialized.html">System.Collections.Specialized</a></td>
+   <td>
+   Contains specialized and strongly-typed collectionsm such as a linked list dictionary, a bit vector, and collections that contain only strings.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.ComponentModel.html">System.ComponentModel</a></td>
+   <td>
+   Contains classes that are used to implement the run-time and design-time behavior of components and controls. This namespace includes the base classes and interfaces for implementing attributes and type converters and binding to data sources.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.ComponentModel.DataAnnotations.html">System.ComponentModel.DataAnnotations</a></td>
+   <td>
+   Contains attribute classes that are used to define metadata for ASP.NET Core data controls. 
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.ComponentModel.DataAnnotations.Schema.html">System.ComponentModel.DataAnnotations.Schema</a></td>
+   <td>
+   Provides support for attribute classes that are used to define metadata for ASP.NET Core data controls.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Data.html">System.Data</a></td>
+   <td>
+   Provides access to classes that efficiently manage data from multiple data sources.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Data.Common.html">System.Data.Common</a></td>
+   <td>
+   Contains classes shared by the .NET Core data providers.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Data.SqlClient.html">System.Data.SqlClient</a></td>
+   <td>
+   The data provider for SQL Server.
```

Comment:
Maybe say "Microsoft SQL Server", to make it clear this is not for generic SQL servers?